api/preprints/views.py

```diff
@@ -82,7 +87,7 @@ class PreprintList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
 
     + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
 
-    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, `provider`, and `subjects`
+    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, and `subjects`
```

Comment:
Filtering by provider name was broken, I removed it instead of fixing it for speed because this was being handled by `/v2/preprint_providers/<provider_id>/preprints/` but happy to fix it instead